runtimes/alibi-explain/tests/helpers/sk_model.py

```diff
@@ -0,0 +1,66 @@
+import os
+from pathlib import Path
+import joblib
+
+from sklearn.ensemble import GradientBoostingClassifier
+from alibi.datasets import fetch_adult
+
+from mlserver import MLModel
+from mlserver.codecs import NumpyCodec
+from mlserver.types import InferenceRequest, InferenceResponse
+
+
+_MODEL_PATH = Path(os.path.dirname(__file__)).parent / ".data" / "sk_income" / "model.joblib"
+
+
+def get_sk_income_model_uri() -> Path:
+    if not _MODEL_PATH.exists():
+        _train_sk_income()
+    return _MODEL_PATH
+
+
+class SKIncomeModel(MLModel):
+    async def predict(self, payload: InferenceRequest) -> InferenceResponse:
+        np_codec = NumpyCodec
+        model_input = payload.inputs[0]
+        input_data = np_codec.decode_input(model_input)
+        output_data = self._model(input_data)
+        return InferenceResponse(
+            model_name=self.name,
+            outputs=[np_codec.encode_output("predict", output_data)]
+        )
+
+    async def load(self) -> bool:
+        self._model = joblib.load(get_sk_income_model_uri())
+        return True
```

Comment:
Do we use this one?

Reply:
Nope. I'd included as anticipating using it when I add black-box explainer tests requiring tabular data (and a model) e.g. `PartialDependence` test etc (similar to the `TFMNISTModel` in `helper/tf_model.py`). But https://github.com/SeldonIO/MLServer/commit/75ab2e4c35343f599e944a3f2816355af833c944 removes it for now, and can add back in later if needed...